src/v2/guide/components-slots.md

```diff
@@ -85,145 +163,396 @@ Para estos casos, el elemento `<slot>` tiene un atributo especial, `name`, el cu
 </div>
 ```
 
-Para proporcionar contenido a los slots nombrados, se puede usar el atributo `slot` en un elemento `<template>` en el padre:
+Un `<slot>` sin `nombre` tiene implícitamente el nombre "default".
+
+Para proporcionar contenido a los slots con nombre, podemos usar la directiva `v-slot` en un `<template>`, proporcionando el nombre del slot como argumento de `v-slot`:
 
 ```html
 <base-layout>
-  <template slot="header">
-    <h1>Aquí podría haber un título de página.</h1>
+  <template v-slot:header>
+    <h1>Aquí podría ir un título de página</h1>
   </template>
 
   <p>Un párrafo para el contenido principal.</p>
-  <p>Y otro más.</p>
+  <p>Y otra más. </p>
 
-  <template slot="footer">
-    <p>Aquí hay alguna información de contacto</p>
+  <template v-slot:footer>
+    <p>Aquí va alguna información de contacto</p>
   </template>
 </base-layout>
 ```
 
-O, el atributo `slot` también se puede usar directamente en un elemento normal:
+Ahora todo lo que está dentro de los elementos de `<template>` se pasará a los slots correspondientes. Cualquier contenido que no esté envuelto en un `<template>` usando `v-slot` se asume que es para el slot por defecto.
 
-``` html
+Sin embargo, todavía puede envolver el contenido del slot por defecto en un `<template>` si quiere ser explícito:
+
+```html
 <base-layout>
-  <h1 slot="header">Aquí podría haber un título de página.</h1>
+  <template v-slot:header>
+    <h1>Aquí podría ir un título de página</h1>
+  </template>
 
-  <p>Un párrafo para el contenido principal.</p>
-  <p>Y otro más.</p>
+  <template v-slot:por defecto>
+    <p>Un párrafo para el contenido principal.</p>
+    <p>Y otra más. </p>
+  </template>
 
-  <p slot="footer">Aquí hay alguna información de contacto</p>
+  <template v-slot:footer>
+    <p>Aquí va alguna información de contacto</p>
+  </template>
 </base-layout>
 ```
 
-Todavía puede haber un slot sin nombre, que es el **slot predeterminado** que sirve como una salida para todos los contenidos no coincidentes. En los dos ejemplos anteriores, el HTML representado sería:
+De cualquier manera, el HTML renderizado será:
 
-``` html
+```html
 <div class="container">
   <header>
-    <h1>Aquí podría haber un título de página.</h1>
+    <h1>Aquí podría ir un título de página</h1>
   </header>
   <main>
     <p>Un párrafo para el contenido principal.</p>
-    <p>Y otro más.</p>
+    <p>Y otra más.</p>
   </main>
   <footer>
-    <p>Aquí hay alguna información de contacto</p>
+    <p>Aquí va alguna información de contacto</p>
   </footer>
 </div>
 ```
 
-## Contenido del slot por defecto
+Note que **`v-slot` sólo puede ser agregado a un `<template>`** (con [una excepción](#Abbreviated-Syntax-for-Lone-Default-Slots)), a diferencia del atributo depreciado [`slot`](#Deprecated-Syntax).
```

Comment:
Typo:
```suggestion
Note que **`v-slot` sólo puede ser agregado a un `<template>`** (con [una excepción](#Abbreviated-Syntax-for-Lone-Default-Slots)), a diferencia del atributo deprecado [`slot`](#Deprecated-Syntax).
```

Reply:
Was a double typo :) I've been using "obsoleto" for "deprecated". I will change it.